### PR TITLE
Case sensitivity of country codes

### LINKED ISSUE #164: Case sensitivity of country codes in country page URLs
## Expected Behavior
Explorer should be able to handle minor typos in URLs where `/country/al` can be treated as `/country/AL` and preferably update the URL shown in the browser.

## Actual Behavior
Currently, only the flag is insensitive to country codes. Rest of the page only renders default content (which itself needs to be handled better).

### PR DESCRIPTION
**Pull request by NidhiKJha**
This pull request fixes #164 
It handles case senstivity in URLs where /country/al can be treated as /country/AL and updates the URL shown in the browser.

### CODE REVIEW

**Comment by NidhiKJha** [comment]
@hellais @sarathms Please review my pull request :)

File: pages/country.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ import AppsSection from '../components/country/Apps'
 import NetworkPropertiesSection from '../components/country/NetworkProperties'
 import { CountryContextProvider } from '../components/country/CountryContext'
 
+
```
Comment (sarathms) [inline pages/country.js:25]: Extra empty line

File: pages/country.js [hunk 2]
```diff
@@ -45,8 +46,19 @@ const AnimatedHeading = styled(Heading)`
 `
 
 export default class Country extends React.Component {
-  static async getInitialProps ({ query }) {
+  static async getInitialProps ({ req, res, query } ) {
```
Comment (sarathms) [inline pages/country.js:49]: Needs a little reformatting. Remove the extra space toward the end `...query })`

File: pages/country.js [hunk 3]
```diff
@@ -45,8 +46,19 @@ const AnimatedHeading = styled(Heading)`
...
     const { countryCode } = query
+
+    if (res && (countryCode !== countryCode.toUpperCase())) {
+      res.writeHead(301, {
+        Location: `/country/${countryCode.toUpperCase()}`
+      });
+
+      res.end();
+      return {};
```
Comment (sarathms) [inline pages/country.js:58]: Remove the semicolons. We don't use them in our coding style.

File: pages/country.js (hunk 1 above)
Comment (NidhiKJha) [reply pages/country.js]: Removed this line

File: pages/country.js (hunk 3 above)
Comment (NidhiKJha) [reply pages/country.js]: Removed

**Comment by NidhiKJha** [comment]
I have checked it. The code works well in the production as expected. Please review this and let me know if I need to make any changes. Thanks!

**Comment by NidhiKJha** [comment]
Thank You. I don't know cypress yet but I'll look into it.

**Pull request merged**